daemon/networkdriver/bridge/driver.go

```diff
@@ -113,6 +114,13 @@ func InitDriver(config *Config) error {
 		addrsv6    []net.Addr
 		bridgeIPv6 = "fe80::1/64"
 	)
+
+	// try to modprobe bridge first
+	// see gh#12177
+	if out, err := exec.Command("modprobe", "bridge", "nf_nat").Output(); err != nil {
+		logrus.Errorf("Running modprobe bridge nf_nat failed with message: %s, error: %v", out, err)
```

Comment:
Is this a kernel version thing because I specifically read the man pages
before doing this

On Wednesday, April 15, 2015, mh0rst <notifications@github.com> wrote:

> In daemon/networkdriver/bridge/driver.go
> <https://github.com/docker/docker/pull/12382#discussion_r28405188>:
>
> > @@ -113,6 +114,13 @@ func InitDriver(config *Config) error {
> >  		addrsv6    []net.Addr
> >  		bridgeIPv6 = "fe80::1/64"
> >  	)
> > +
> > +	// try to modprobe bridge first
> > +	// see gh#12177
> > +	if out, err := exec.Command("modprobe", "bridge", "nf_nat").Output(); err != nil {
> > +		logrus.Errorf("Running modprobe bridge nf_nat failed with message: %s, error: %v", out, err)
>
> Use the -a (insert all) option.
>
> $ sudo modprobe -v -a test1 test2
> libkmod: ERROR ../libkmod/libkmod.c:505 kmod_lookup_alias_from_builtin_file: could not open builtin file '/lib/modules/2.6.32-042stab092.3/modules.builtin.bin'
> WARNING: Module test1 not found.
> libkmod: ERROR ../libkmod/libkmod.c:505 kmod_lookup_alias_from_builtin_file: could not open builtin file '/lib/modules/2.6.32-042stab092.3/modules.builtin.bin'
> WARNING: Module test2 not found.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12382/files#r28405188>.
>